File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.py
import sys

n, m = map(int, sys.stdin.readline().split())

list = []

for i in range(n + 1):
    list.append(i)

for a in range(m):
    i, j = map(int, sys.stdin.readline().split())
    tmp = list[i]
    list[i] = list[j]
    list[j] = tmp

print(*list[1:])

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.js
const rl = require('readline').createInterface({
    input: process.stdin,
    output: process.stdout,
});

let n = 0;
let m = 0;
let baskets = [];

rl.on('line', (line) => {
    if (n === 0) {
        [n, m] = line.split(' ').map(Number);
        baskets = Array.from({ length: n }, (_, index) => index + 1);
    } else {
        const [i, j] = line.split(' ').map(Number);
        [baskets[i - 1], baskets[j - 1]] = [baskets[j - 1], baskets[i - 1]];

        m--;

        if (m === 0) {
            rl.close();
        }
    }
}).on('close', () => {
    console.log(baskets.join(' '));
    process.exit(0);
});